src/main/webapp/app/courses/results/result.component.ts

```diff
@@ -32,13 +23,13 @@ export class SafeHtmlPipe implements PipeTransform {
 export class ResultComponent implements OnInit, OnChanges, OnDestroy {
 
     @Input() participation: Participation;
-    // @Output() onNewResult;
+    @Input() building: boolean;
+    @Output() newResult = new EventEmitter<object>();
```

Comment:
We have result, results and newResult now. This might be too many result variables